Production/python/customiseHLT.py

```diff
@@ -0,0 +1,187 @@
+
+  process.tauTable = cms.EDProducer("SimpleCandidateFlatTableProducer",
+    src = cms.InputTag( "hltHpsPFTauProducer" ),
+    cut = cms.string(""),
+    name= cms.string("Tau"),
+    doc = cms.string("HLT taus"),
+    singleton = cms.bool(False), # the number of entries is variable
+    extension = cms.bool(False), # this is the main table
+    variables = cms.PSet(
+      P4Vars,
+      charge = Var("charge", int, doc="electric charge"),
+    )
+  )
+  # cms.EDProducer : an object that produces a new data object
+
+  process.pfCandTable = cms.EDProducer("SimpleCandidateFlatTableProducer",
+    src = cms.InputTag( "hltParticleFlowForTaus" ),
+    cut = cms.string(""),
+    name= cms.string("PFCand"),
+    doc = cms.string("HLT PF candidates for taus"),
+    singleton = cms.bool(False), # the number of entries is variable
+    extension = cms.bool(False), # this is the main table
+    variables = cms.PSet(
+      P4Vars,
+      charge = Var("charge", int, doc="electric charge"),
+
+      vx = Var("vx", float, doc='x coordinate of vertex position'),
+      vy = Var("vy", float, doc='y coordinate of vertex position'),
+      vz = Var("vz", float, doc='z coordinate of vertex position'),
+      vertexChi2 = Var("vertexChi2", float, doc='chi-squares'),
+      vertexNdof = Var("vertexNdof", float, doc='Number of degrees of freedom,  Meant to be Double32_t for soft-assignment fitters'),
+      pdgId = Var("pdgId", int, doc='PDG identifier'),
+      status = Var("status", int, doc='status word'),
+      longLived = Var("longLived", bool, doc='is long lived?'),
+      massConstraint = Var("massConstraint", bool, doc='do mass constraint?'),
+      # source: cmssw/PhysicsTools/NanoAOD/plugins/SimpleFlatTableProducerPlugins.cc 
+      trackDz = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dz : -999.", float, doc = "track Dz"),
+      trackDxy = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dxy : -999.", float, doc = "track Dxy"),
+      trackIsValid = Var("trackRef.isNonnull && trackRef.isAvailable", bool, doc = "track is valid"),
+      
+      trackDzError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dzError : -999.", float, doc = "track DzError"),
+      trackDxyError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dxyError : -999.", float, doc = "track DxyError"),
+      
+      trackPt = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.pt : -999.", float, doc = "track Pt"),
+      trackEta = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.eta : -999.", float, doc = "track Eta"),
+      trackPhi = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.phi : -999.", float, doc = "track Phi"),
+      trackPtError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.ptError : -999.", float, doc = "track PtError"),
+      trackPtaError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.etaError : -999.", float, doc = "track PtaError"),
```

Comment:
small typo, should be `trackEtaError`
```suggestion
      trackEtaError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.etaError : -999.", float, doc = "track PtaError"),
```